build-gradle-plugin/src/main/kotlin/cz/ackee/gradle/AckeePluginKotlin.kt

```diff
@@ -239,7 +242,8 @@ class AckeePluginKotlin : Plugin<Project> {
 
                 maybeCreate("release").apply {
                     keyAlias = keystoreProperties["key_alias"] as String?
-                    storeFile = File(keystoreDir, keystoreProperties["key_file"] as String?)
+                    val keystoreFile = File(keystoreDir, keystoreProperties["key_file"] as String? ?: "")
```

Comment:
what is the advantage of having it as a separate property?

Reply:
I don't like having to read a constructor parameter that has to be cast with a null catch. I reformatted it in a rebase to be
```kotlin
maybeCreate("release").apply {
    keyAlias = keystoreProperties["key_alias"] as String?
    storePassword = keystoreProperties["key_password"] as String?
    keyPassword = keystoreProperties["key_password"] as String?

    val keystoreFileName = keystoreProperties["key_file"] as String? ?: ""
    storeFile = File(keystoreDir, keystoreFileName)
}
```

Reply:
ok, like previously, i don't find it necessary but i will not block your improvements :D